requirements/requirements-testing.txt

```
@@ -5,3 +5,4 @@ pytest-django>=4.5.2,<5.0
 importlib-metadata<5.0
 # temporary pin of attrs
 attrs==22.1.0
+pytz
```

Should we add a note here, similar to

> "Django 5.0 has removed pytz; this test should eventually be able to get removed."

from the skip reason?

---

Sure, we can, won't hurt. How about this?

```suggestion
pytz  # Remove when dropping support for Django<5.0
```

However, if I was making the change to drop support for Django<5.0, I'd probably remove the skip from the test and forget about this file... Would help to clean it up later in case we forget though 👍🏻